docs/rescal-snow-tutorial.md

```diff
@@ -119,8 +119,12 @@ You can run an example snow feature with sintering with:
 ```bash
 cd scripts
 ./sintering.run
```

Comment:
@kellykochanski I think you're missing a command here, or forgot to add your changes to the .run script?

Reply:
If you still have files around from `snow_cone.run`, `sintering.run` will prompt you to delete them, without requiring the user to run an extra command.

Reply:
Clarifying in tutorial:


You can run an example snow feature with sintering with:
```bash
cd scripts
./sintering.run
```

As we are running all the tutorials in the same `scripts` directory, it's important to delete old output to avoid mixing the output from two runs. The `sintering.run` command should prompt you to do this:
```bash
Deleting files of types *.(png|bin|csp|log|gz|data|vel|car) in current directory [n|y] and out directory ?
y
```
Or you can do it manually with `./clean`.

Reply:
Looks good. I missed the commit changing the behavior of the clean script, which is getting invoked in the `.run` scripts.